nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Welcome page

# Description
Home Page set up with welcome message "Good morning/afternoon/evening user" 
May Fix # (57) Tag latency. need more observation to be certain
## Type of change
Please delete options that are not relevant.
# Testing Instructions
All tags should update reliably now. Tag Management Add/Delete/Update as well as adding tags to a post from the post details page.
Welcome page. When the user is logged in the welcome page will display a message of "Good {Morning/Afternoon/Evening} {userName}" as well as a user avatar. Three random {at this point} posts will be displayed. If the image does not successfully load, a default image of a stack of newspapers will display in place of the stock broken image link.
# Checklist:
